internal/operators/ocs/validations.go

```diff
@@ -196,6 +195,21 @@ func gbToBytes(gb int64) int64 {
 	return gb * int64(units.GB)
 }
 
+// count all disks of drive type ssd or hdd
+func getDisks(disks []*models.Disk) int {
```

Comment:
Maybe call it getDisksCount or similar?

Reply:
To make it even more specific - getValidOCSDiskCount()